Assignment5Classes/src/retailclasses/Store.java

```diff
@@ -0,0 +1,90 @@
+
+package retailclasses;
+
+import java.util.Scanner;
+
+/**
+ * Program to store item information for RetailItem
+ * @author Seth Frosch
+ */
+public class Store {
+    
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        /**
+         * @param description A string field for a description of the item
+         * @param unitsOnHand An integer for the amount of an item in the 
+         * inventory
+         * @param price A double for the price of the item
+         * @param keyboard Will accept user input
+         */
+        String description;
+        int unitsOnHand;
+        double price;
+        Scanner keyboard = new Scanner(System.in);
+        
+        /**
+         * Outputs prompts for user to enter description, amount on hand, and
+         * price for item 1
+         */
+        System.out.println("Enter the description for item 1:");
+        description = keyboard.nextLine();
```

Comment:
Continuous reassignment of these variables is a little risky. There is a principle called DRY (don't repeat yourself). These sections are a prime example of where some abstraction can clean up these repeated sections and assure that the logic is contained and reduce side effects.